src/GitHub.App/ViewModels/PullRequestFileViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestFileViewModel : IPullRequestFileViewModel
```

Comment:
I don't see anything that's specific to this view in particular. It's just a representation of a file change in a pull request. It has no logic, only data, usable by any view or viewmodel to do stuff. Screams of model to me 😄